sdk/formrecognizer/test-resources.json

```diff
@@ -74,10 +70,6 @@
             "type": "string",
             "value": "[variables('endpointValue')]"
         },
-        "FORM_RECOGNIZER_BLOB_CONTAINER_SAS_URL": {
```

Comment:
I might be missing something here. How are we then setting this property so that it is available for our tests?

Reply:
Sorry about it, I send another [PR ](https://github.com/Azure/azure-sdk-for-net/pull/12862)to re-add this parameter.

Reply:
Thank you!